src/js/components/Ballot/PollingPlaceLocator.jsx

```diff
@@ -23,7 +25,7 @@ export default class PollingPlaceLocator extends Component {
             <div className="row intro-modal__grid intro-modal__default-text u_margin-center">
               <div className="container-fluid u-inset--md">
                 <div>
-                  <iframe title="voter-information-project" style={iframeStyle} src="https://tool.votinginfoproject.org/iframe-embed.html" />
+                  {/* <iframe title="voter-information-project" style={iframeStyle} src="https://tool.votinginfoproject.org/iframe-embed.html" /> */}
```

Comment:
This iframe code is needed to show the best available Polling Place Locator code. We will be placing this somewhere in the new Vote section once it is designed. There is also a dedicated polling location locator route that Cordova uses that @SailingSteve set up.